src/lava_callback.py

```diff
@@ -211,6 +210,23 @@ def async_job_submit(api_helper, node_id, job_callback):
     job_result = job_callback.get_job_status()
     device_id = job_callback.get_device_id()
     storage_config_name = job_callback.get_meta('storage_config_name')
+    input_file_types = [
+      'nfsrootfs',
+      'rootfs',
+      'ramdisk',
+      'initrd',
+      'ndbroot',
+      'persistent_nfs'
+    ]
```

Comment:
I think this could be a constant at the top of this file, if possible with a comment explaining how this values were created/selected.